File: 151004/Bosko/lab2/lab2/DTO/NoteRequestTo.cs
using lab2.DTO.Interface;

namespace lab2.DTO
{
    public class NoteRequestTo(int Id, int? NewsId, string? Content) : IRequestTo
    {
        public int Id { get; set; } = Id;

        public int? NewsId { get; set; } = NewsId;

        public string? Content { get; set; } = Content;
    }
}

File: 151003/Rustamov/publisher/src/main/java/org/example/publisher/impl/note/dto/NoteRequestTo.java
package org.example.publisher.impl.note.dto;

import jakarta.validation.constraints.Size;
import lombok.AllArgsConstructor;
import lombok.Data;

import java.math.BigInteger;

@Data
@AllArgsConstructor
public class NoteRequestTo {
    private BigInteger id;
    private BigInteger newsId;
    @Size(min = 2, max = 2048)
    String content;
}
